### ISSUE TITLE
Loss for reward model in summarize_rlhf example is not normalized correctly. 

### ISSUE THREAD

**Issue by exfoxz** [CarperAI/trlx#220]
Hello there,
Thanks for the great example of summarize_rlhf. It looks to me the loss for reward model in `reward_model.py` is not normalized correctly. Specifically, `loss = loss / (bs)` is called in the the loop through items in each batch, repeatedly. Should it be outside? Thanks.

https://github.com/CarperAI/trlx/blob/2f230017a74448e16939a39f5e198cc8aa2eece9/examples/summarize_rlhf/reward_model/reward_model.py#L97

**Comment by PhungVanDuy** [comment]
Thank you for your comment this is a typo issue when we refactor the code. I will update the fix now.
